File: app/src/main/java/ostmodern/skylark/repository/remote/SkylarkClient.java
package ostmodern.skylark.repository.remote;

import io.reactivex.Observable;
import retrofit2.http.GET;


public interface SkylarkClient {

    @GET("/api/sets/")
    Observable<SetApiResponse> getSetList();

    @GET("/api/images/")
    Observable<ImageApiResponse> getImageList();
}

File: app/src/main/java/ostmodern/skylark/repository/remote/SkylarkClient.kt
package ostmodern.skylark.repository.remote

import io.reactivex.Observable
import retrofit2.http.GET


interface SkylarkClient {

    @get:GET("/api/sets/")
    val setList: Observable<SetApiResponse>

    @get:GET("/api/images/")
    val imageList: Observable<ImageApiResponse>
}
